api/AudioData.json

```diff
@@ -14,9 +14,11 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
+            "version_added": "130"
+          },
+          "firefox_android": {
             "version_added": false
```

Comment:
I'm noticing that all of these features are being set as unsupported in Firefox Android.  This seems very unlikely.  These should all stay as "mirror", right?

Reply:
This comes from @hamishwillee's research in https://github.com/mdn/browser-compat-data/pull/24274

Reply:
I can't generally talk about features, but any related to WebCodecs were released in 130 but were "preview only" in Android. Which is false. See  https://github.com/mdn/browser-compat-data/pull/24274

Reply:
Ah, the joy of overlap.

Reply:
It is still true in current source. See https://searchfox.org/mozilla-central/source/modules/libpref/init/StaticPrefList.yaml#3180

Reply:
I'm seeing that in the [bug linked](https://bugzilla.mozilla.org/show_bug.cgi?id=1908572) in that PR, Android was indeed disabled.  It seemed strange at first (since it was all of the features touched in this PR), but it checks out!